fiat-github/src/main/java/com/netflix/spinnaker/fiat/config/GitHubConfig.java

```diff
@@ -79,13 +74,16 @@ public void log(String message) {
 
   private static class BasicAuthRequestInterceptor implements RequestInterceptor {
 
-    @Getter
     @Setter
     private String accessToken;
 
     @Override
     public void intercept(RequestFacade request) {
-      request.addQueryParam("access_token", accessToken);
+      // Although the encoded string is normally "user:password" for Basic auth, GitHub appears to
+      // only look at the value to the right of the colon, or the whole value if no colon exists.
+      // This is nice for us, because we don't necessarily have a username to pass along anyway.
+      String encoded = Base64Utils.encodeToString(accessToken.getBytes(StandardCharsets.UTF_8));
+      request.addHeader("Authorization", "Basic " + encoded);
```

Comment:
According the github doc it may be wrong : https://developer.github.com/v3/#authentication You might replace `Basic`  by `token`